iron-overlay-backdrop.html

```diff
@@ -109,9 +114,14 @@
      * Hides the backdrop if needed.
      */
     close: function() {
+      // Always update z-index
+      this.style.zIndex = this._manager.backdropZ();
       // only need to make the backdrop invisible if this is called by the last overlay with a backdrop
       if (this._manager.getBackdrops().length < 2) {
         this._setOpened(false);
+        // complete() will be called after the transition is done.
+        // If animations are disabled via custom-styles, user is expected to call
+        // complete() after close()
```

Comment:
@cdata potentially we could have a property for the backdrop `noAnimation`, should I do it here or in a separate PR?